src/task_scl_eff_pot_hab.py

```diff
@@ -206,38 +261,76 @@ def dilate(image, distance):
 
         scl_polys = potential_core.add(potential_stepping_stone).selfMask()
 
-        # Temp geometry for RFE scenario
-        # geometry = ee.Geometry.Polygon(
-        #     [
-        #         [
-        #             [129.28241561583914, 51.490498638741165],
-        #             [129.28241561583914, 42.20681310459823],
-        #             [140.35663436583914, 42.20681310459823],
-        #             [140.35663436583914, 51.490498638741165],
-        #         ]
-        #     ],
-        #     None,
-        #     False,
-        # )
+        # TODO: the probability threshold process will change with final version of probability
+        probability_thresh = (
+            self.probability.add(1).gte(self.thresholds["probability"]).selfMask()
+        )
+
+        # TODO: incorporate actual survey effort input
+        survey_effort = ee.Image.constant(1)
 
         scl_polys = (
             ee.Image(1)
             .updateMask(scl_polys)
             .addBands(scl_polys)
-            .addBands(self.probability)
-            .rename(["constant", "scl_poly", "probability"])
+            .addBands(probability_thresh)
+            .addBands(survey_effort)
+            .rename(["scl_poly", "size", "probability", "effort"])
             .reduceToVectors(
-                reducer=ee.Reducer.max(),
-                geometry=self.extent,
+                reducer=ee.Reducer.max(),  # TODO: need to consider reducer for each band to delineate polygons
+                geometry=ee.Geometry.MultiPolygon(self.aoi).bounds(),
```

Comment:
`ee.Geometry.MultiPolygon(self.aoi, proj=self.crs, geodesic=False)`
or, couldn't we just do 
`ee.Geometry.Polygon(self.extent)`
?